node/src/components/sync_leaper.rs

```diff
@@ -263,6 +263,7 @@ impl SyncLeaper {
     where
         REv: From<FetcherRequest<SyncLeap>> + Send,
     {
+        info!(%sync_leap_identifier, "registering leap attempt");
```

Comment:
why do we need this at info?

Reply:
I find it useful to delimitate in the logs when a sync leap is registered in order to track the progress afterwards.